### ISSUE TITLE
contrib-spoor: failing to track SCORM details in IE8

### ISSUE THREAD

**Issue by ian-dowhile** [adaptlearning/adapt_framework#1767]
When running a course using IE8 in an LMS, the course is failing to commit its tracking data back to the LMS.

I've found that the following commit added a call to document.removeEventListener when the browser window is being unloaded.  As that method isn't supported by IE8, it was throwing a JavaScript error and the subsequent call to scorm.finish() was not being made.

https://github.com/adaptlearning/adapt-contrib-spoor/commit/df4817cabe6f09d91658202bf3f7b439f21f81d0

I would suggest that in the function removeEventListeners, this:

```
document.removeEventListener("visibilitychange", this.onVisibilityChange);
```

Is replaced with:

```
if (document.removeEventListener) {
   document.removeEventListener("visibilitychange", this.onVisibilityChange);
}
```

**Comment by brian-learningpool** [comment]
Hi @ian-dowhile, that makes sense.  

Can you generate a PR?

**Comment by brian-learningpool** [comment]
@ian-dowhile, thanks for raising this.  I've made a PR here if you want to test:
https://github.com/adaptlearning/adapt-contrib-spoor/pull/162

**Comment by brian-learningpool** [comment]
Fixed in v2.1.3 of Spoor.
